### DISCUSSION TITLE
LD24AJTA  Trimpot

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#27] [Q&A] [answered]
I've adjusted my 12v DC power source to be exactly 12.5V.

Based on #24 It appears the RCS resistor needs to be swapped.

Can you please provide more data on the package type, power rating, etc this resistor should be?  At mouser I am only finding current sensing resistors in the 1206 package.  I am less versed in SMD component / pcb builds so any guidance would be helpful.

Once this is done, does that mean the trimpot then gets adjusted to the maximum current output?

**Comment by srozum** [comment, marked as answer]
When driver controlled by PWM signal the trimpot does nothing. So leave it alone.

Calibration procedure described here: https://github.com/srozum/film_camera_tester/wiki/Light-Source#calibration
